From a8adcdd75558dae9697e11a2f93ff85d358217fc Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 13 Mar 2023 23:22:27 +0000
Subject: [PATCH 10/17] gnu: cl-gamepad: Update to 3.0.0-2.647f6ee.

* gnu/packages/lisp-xyz.scm (sbcl-cl-gamepd): Update to 3.0.0-2.647f6ee.
  [arguments]{phases}: Use search-input-file in patch-evdev-path phase.
  [inputs]: Remove labels.
---
 gnu/packages/lisp-xyz.scm | 19 +++++++++----------
 1 file changed, 9 insertions(+), 10 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 19e20a26f9..a7b7374e0d 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -20733,8 +20733,8 @@ (define-public cl-sdl2-ttf
   (sbcl-package->cl-source-package sbcl-sdl2-ttf))
 
 (define-public sbcl-cl-gamepad
-  (let ((commit "7e12137927b42db064ffbf9ea34bd4790ad4bb33")
-        (revision "1"))
+  (let ((commit "647f6ee8f40048286d743d79845c3753fba9d8f1")
+        (revision "2"))
     (package
       (name "sbcl-cl-gamepad")
       (version (git-version "3.0.0" revision commit))
@@ -20746,7 +20746,7 @@ (define-public sbcl-cl-gamepad
                (commit commit)))
          (file-name (git-file-name "cl-gamepad" version))
          (sha256
-          (base32 "1gzx590i7s81qmramnjvfzrrq5yppas8yxqq1jl3yzqhhjwjfvkd"))))
+          (base32 "0w9lcahgqacc39932jp2ghid9sl4wg4vyaza8vdnghmixdl49cin"))))
       (build-system asdf-build-system/sbcl)
       (arguments
        `(#:phases
@@ -20754,14 +20754,13 @@ (define-public sbcl-cl-gamepad
            (add-after 'unpack 'patch-evdev-lib-path
              (lambda* (#:key inputs #:allow-other-keys)
                (substitute* "evdev-cffi.lisp"
-                 (("libevdev.so" all)
-                  (string-append (assoc-ref inputs "libevdev")
-                                 "/lib/" all))))))))
+                 (("libevdev.so")
+                  (search-input-file inputs "/lib/libevdev.so"))))))))
       (inputs
-       `(("cffi" ,sbcl-cffi)
-         ("documentation-utils" ,sbcl-documentation-utils)
-         ("libevdev" ,libevdev)
-         ("trivial-features" ,sbcl-trivial-features)))
+       (list libevdev
+             sbcl-cffi
+             sbcl-documentation-utils
+             sbcl-trivial-features))
       (home-page "https://shirakumo.github.io/cl-gamepad/")
       (synopsis "Library for access to gamepads and joystick input devices")
       (description
-- 
2.39.2

